File: Practice(Beginner)/C++/Primality Test.cpp
// Source: https://www.codechef.com/problems/PRB01

#include <bits/stdc++.h>
using namespace std;

bool isPrime(int n){
    for (int i = 2; i < (int)(pow(n,0.5)+1); ++i) {
        if(n%i == 0){
            return false;
        }
    }
    return true;
}

int main() {
    int t, n;
    cin >> t;
    while(t--){
        cin >> n;
        isPrime(n) ? cout << "yes" << endl : cout << "no" << endl;
    }
    return 0;
}

File: Practice(Beginner)/Python3/Primality_Test.py
# Source: https://www.codechef.com/problems/PRB01

def isPrime(num):
    for i in range(2,int(num**0.5)):
        if(num%i == 0):
            return False
    return True

n = int(input())
for i in range(n):
    x = int(input())
    if(isPrime(x)):
        print("yes")
    else:
        print("no")
